<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormSubKategori</class>
 <widget class="QWidget" name="FormSubKategori">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Simpan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>150</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Hapus</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="formGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>371</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="idSubKetegoriLabel">
      <property name="text">
       <string>Id Sub Ketegori</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="idSubKetegoriLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="namaSubKategoriLabel">
      <property name="text">
       <string>Nama Sub Kategori</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="namaSubKategoriLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="idKategoriLabel">
      <property name="text">
       <string>Id Kategori</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="idKategoriLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QTableView" name="tableSubkategori">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>371</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
